### ISSUE TITLE
clustering reproduce question

### ISSUE THREAD

**Issue by sakura8999** [meettyj/HGMAE#2]
Hello, I conducted the clustering experiments on the DBLP dataset , but the scores are lower than the results reported in your paper.
The best epoch is: 293[clustering] nmi: [.7192, .92] ari: [.768, .092]
 Are there any other factors I should pay attention to besides the parameters you recommended? thank you.

<img width="409" alt="9607174b0f02aff079ad6deaa265418" src="https://github.com/meettyj/HGMAE/assets/127652105/9f16caed-a007-419d-8ed3-3760529cd7cf">

**Comment by MengYingDai-SHU** [comment]
I ran the code on a 4090 graphics card

**Comment by jhqi** [comment]
> Hello, I conducted the clustering experiments on the DBLP dataset , but the scores are lower than the results reported in your paper. The best epoch is: 293[clustering] nmi: [.7192, .92] ari: [.768, .092] Are there any other factors I should pay attention to besides the parameters you recommended? thank you.
> 
> <img alt="9607174b0f02aff079ad6deaa265418" width="409" src="https://private-user-images.githubusercontent.com/127652105/271776800-9f16caed-a007-419d-8ed3-3760529cd7cf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE3NzcwMjAsIm5iZiI6MTcwMTc3NjcyMCwicGF0aCI6Ii8xMjc2NTIxMDUvMjcxNzc2ODAwLTlmMTZjYWVkLWEwMDctNDE5ZC04ZWQzLTM3NjA1MjljZDdjZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIwNVQxMTQ1MjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zN2U5YjYwYjgwYTY5YTUyZTJiZjZjMWY1YzZiZWVjOGRkMWE3MDk0MDFkZTMzYjM3ZDc3M2EzNWU3ZTJmMWVlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.OmlUjaMojPb5ZvQ_nVyVzPxaJKaaXdaOGXnUwlSkA98">

hi, I've also reproduced the clustering task on DBLP. I directly used the code given by the author below:
python main.py --dataset dblp --task clustering --use_cfg 

and I got a quite poor result, even much lower than yours. Is there anything I need to modify on the author's code? plz

I didn't do anything to the source code. And it seems that the author has fixed the random seed in the code. But my training state is different from yours:
![微信截图_20231205195524](https://github.com/meettyj/HGMAE/assets/104731032/1c81a062-d825-4469-9ddf-f1997a6d90dc)

**Comment by sakura8999** [comment]
I Sorry, I was a little late to see the email. I didn't change the author's code when I reproduced, and I did it on the RTX4090. Maybe you might want to consider changing graphics cards.

2023-12-05 19:49:19 "qqqck18" ***@***.***> 写道：

Hello, I conducted the clustering experiments on the DBLP dataset , but the scores are lower than the results reported in your paper. The best epoch is: 293[clustering] nmi: [.7192, .92] ari: [.768, .092] Are there any other factors I should pay attention to besides the parameters you recommended? thank you.

hi, I've also reproduced the clustering task on DBLP. I directly used the code given by the author below:
python main.py --dataset dblp --task classification --use_cfg

and I got a quite poor result, even much lower than yours. Is there anything I need to modify on the author's code? plz

**Comment by zyq-bupt** [comment]
<img width="566" alt="截屏2024-09-11 17 26 40" src="https://github.com/user-attachments/assets/7ee5da3f-4598-480b-aa21-3a39b6806acb">
 I get this. It's higher than yours but still lower than that reported in the paper. I use the default parameters.
Here is my device configuration, maybe it will be useful to you：4090*1, pytorch==2.1.0， cuda==12.1
